css/index.css

```diff
@@ -172,4 +172,71 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
-}
\ No newline at end of file
+}
+
+
+@media (max-width: 768px){
+  body {
+    width: 100%;
+    height: auto;
+    padding: 30px;
+    background:grey;
+  }
+  
+  .container {
+    width: auto;   
+}
+
+.cta .cta-text {
+    margin: 0px;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+embed,iframe,object,video {
```

Comment:
There shouldn't need to be a reason where you need to alter any of these particular elements.